src/components/button.js

```diff
@@ -4,27 +4,33 @@ import { Link } from '../routes';
 import PropTypes from 'prop-types';
 import ClassNames from 'classnames';
 
-const Button = ({ color, icon, label, route }) => {
+const Button = ({ color, icon, label, onClick, route }) => {
     const buttonClass = ClassNames({
         btn: true,
         [color]: color,
     });
 
-    return (
+    return route ? (
```

Comment:
No need for ternary here. 

You should do if() { return ... } ..... return  ... instead